.github/workflows/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,19 @@
+<!--This template is based on https://github.com/macports/macports-ports/blob/master/.github/PULL_REQUEST_TEMPLATE.md, thanks Gcenx!-->
+#### Description
+
+<!--Note: it is best to make pull requests from a branch rather than from main-->
+
+###### Type(s)
+
+- [ ] game page addition
+- [ ] game page modification
+- [ ] other wiki edit
+
+###### Verification <!-- (delete not applicable items) -->
+Have you
+
+- [ ] Followed the [contribution guidelines?](https://github.com/Whisky-App/whisky-book?tab=readme-ov-file#please-read-all-steps-before-contributing)
+- [ ] Ran `./scripts/generate.mjs` followed by `./scripts/lint.mjs`?
```

Comment:
As you have mentioned on Discord, people tend to forget about running a linting script. So I would prefer to split it into two bullet points with each script. It may look redundant but it may ensure more completed lint checks as it will be a "separate" step to do.

Reply:
@mattmaniak the linting script is not necessary if you do everything right.

Reply:
Yes but some people forget about doing things right or they are not aware of some stuff, especially when doing a contribution for a first time. This is why I propose do make lint step even more explicit.

Reply:
agree with matt, resolved